clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -334,49 +353,56 @@ void zoo() {
   // expected-error@+1{{duplicate Intel FPGA loop attribute 'loop_count'}}
   [[intel::loop_count(2)]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
+
+  [[intel::fpga_pipeline(1)]]
+  // expected-error@+1{{duplicate Intel FPGA loop attribute 'fpga_pipeline'}}
+  [[intel::fpga_pipeline(1)]] for (int i = 0; i != 10; ++i)
+    a[i] = 0;
 }
 
 // Test for Intel FPGA loop attributes compatibility
 void loop_attrs_compatibility() {
   int a[10];
-  // no diagnostics are expected
-  [[intel::disable_loop_pipelining]]
-  [[intel::loop_coalesce]] for (int i = 0; i != 10; ++i)
-      a[i] = 0;
-  // expected-error@+3 {{'max_interleaving' and 'disable_loop_pipelining' attributes are not compatible}}
+  [[intel::fpga_pipeline]] [[intel::loop_coalesce]] for (int i = 0; i != 10; ++i)
+    a[i] = 0;
+  // expected-error@+4 {{'max_interleaving' and 'disable_loop_pipelining' attributes are not compatible}}
   // expected-note@+1 {{conflicting attribute is here}}
-  [[intel::disable_loop_pipelining]]
+  [[intel::disable_loop_pipelining]] // expected-warning {{attribute 'intel::disable_loop_pipelining' is deprecated}} \
```

Comment:
I don't think we need so many tests testing deprecation warning. IMO one should be enough. Having these warnings makes it difficult to keep track of what the original intention of the test was - which is in this case is conflicting attributes.

We can also then just remove some tests if this causes duplicate tests with new tests you added below. I assume the mutually excluded attributes will be same for `[[intel::disable_loop_pipelining]]` and `intel::fpga_pipeline`.